articles/search/search-filters.md

```diff
@@ -102,7 +105,7 @@ The following examples illustrate several usage patterns for filter scenarios. F
 + Combination of query string and **$filter**, where the filter creates the subset, and the query string provides the term inputs for full text search over the filtered subset. Using a filter with a query string is the most common usage pattern.
 
    ```
-   search=ooms/any(room: room/BaseRate ge 60 and room/BaseRate lt 300) and Address/City eq 'Seattle'
+   search=Rooms/any(room: room/BaseRate ge 60 and room/BaseRate lt 300) and Address/City eq 'Seattle'
```

Comment:
Per previous comment, this is still missing the search term, and it's still missing the filter.

This query returns all 50 hotels not just the ones in Seattle.  It's missing the filter statement and the search term. If you add "walking distance theaters" as the search term, then you get 3 hotels, with search scores, with the hotel with that phrase at the top.

Here is the string again:

search=walking distance theaters&$filter=Rooms/any(room: room/BaseRate ge 60 and room/BaseRate lt 300) and Address/City eq 'Seattle'&$count=true

Reply:
@HeidiSteen Get the same results as you, hence updated it with this query!